Src/PoissonRecon.h

```diff
@@ -510,8 +511,8 @@ template<typename Real>
 void PoissonRecon<Real>::evaluate()
 {
     m_profiler.start();
-    double valueSum = 0;
```

Comment:
Think I changed this out of concern for precision loss over the sums.  I'm assuming you changed this back to eliminate a precision warning?  Perhaps a cast is in order?  Thoughts?